fe1-web/src/features/evoting/reducer/__tests__/ElectionReducer.test.ts

```diff
@@ -77,20 +72,14 @@ describe('ElectionReducer', () => {
           [mockElectionState.id]: mockElectionState2,
         },
         allIds: [mockElectionState.id],
+        defaultQuestions: [EMPTY_QUESTION],
       } as ElectionReducerState);
     });
 
-    it('throws an error when trying to update an inexistent election', () => {
-      expect(() =>
-        electionReduce(
-          {
-            byId: {},
-            allIds: [],
-          } as ElectionReducerState,
-          updateElection(mockElectionState),
-        ),
-      ).toThrow();
-    });
+    expect(() =>
+      electionReduce(defaultElectionReducerState, updateElection(mockElectionState)),
+    ).toThrow();
+    it('throws an error when trying to update an inexistent election', () => {});
```

Comment:
I suppose these lines 3 lines of test should be in the test 'throws an error when trying to update an inexistent election'?

Reply:
Nice catch. This is indeed an error.